README.md

```diff
@@ -3,106 +3,117 @@
 
-6. Run the migrations.
-
-   ```
-   python manage.py db upgrade
+   ```bash
+   ngrok http 5000
    ```
 
-7. Start the development server.
+   Keep in mind that our endpoint is:
 
    ```
-   python manage.py runserver
+   http://<your-ngrok-subdomain>.ngrok.io/confirm
    ```
 
-8. Check it out at [http://localhost:5000](http://localhost:5000)
+1. You will need to configure Twilio to send requests to your application when SMS are received. You will need to provision at least one Twilio number with sms capabilities so the application's users can make property reservations. On [Twilio Account Settings](#twilio-account-settings) you could find the link to buy a Twilio number. Once you have a Twilio number you need to configure your number to work with your application. Open the number management page and open a number's configuration by clicking on it. Remember that the number where you change the SMS webhook must be the same one you set on the TWILIO_NUMBER setting.
+
+   ![Configure Messaging](webhook.png)
 
 
 That's it!
 
-## Run the tests
+## Tests
 
 You can run the tests locally through [coverage](http://coverage.readthedocs.org/):
 
-    ```
-    $ coverage run manage.py test
-    ```
+```bash
+coverage run manage.py test
+```
 
 You can then view the results with `coverage report` or build an HTML report with `coverage html`.
 
-## Meta
+### Cloud deployment
+
+Additionally to trying out this application locally, you can deploy it to a variety of host services. Here is a small selection of them.
+
+Please be aware that some of these might charge you for the usage or might make the source code for this application visible to the public. When in doubt research the respective hosting service first.
+
+| Service                           |                                                                                                                                                                                                                           |
+| :-------------------------------- | :------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| [Heroku](https://www.heroku.com/) | [![Deploy](https://www.herokucdn.com/deploy/button.svg)](https://heroku.com/deploy)                                                                                                                                       |
+
+## Resources
+
+- The CodeExchange repository can be found [here](https://github.com/twilio-labs/code-exchange/).
+
+## Contributing
+
+This template is open source and welcomes contributions. All contributions are subject to our [Code of Conduct](https://github.com/twilio-labs/.github/blob/master/CODE_OF_CONDUCT.md).
+
+## License
+
+[MIT](http://www.opensource.org/licenses/mit-license.html)
+
+## Disclaimer
+
+No warranty expressed or implied. Software is as is.
```

Comment:
Are there plans to have more Docker specific instructions in the README?

Otherwise, it works great!

Reply:
Hey @bharatmurali! Yes, we will add instructions for Docker.